com.unity.render-pipelines.core/Runtime/Common/DynamicResolutionHandler.cs

```diff
@@ -402,14 +482,19 @@ public Vector2Int GetScaledSize(Vector2Int size)
 
         /// <summary>
         /// Applies to the passed size the scale imposed by the dynamic resolution system.
+        /// This function uses the internal resolved scale from the DRS system.
```

Comment:
I would not use acronyms in the doc here (DRS vs Dynamic Resolution Handler)

Reply:
switched to full name instead of acronym